lib/native-client.js

```diff
@@ -106,6 +106,56 @@ class NativeClient {
   }
 
   /**
+   * Get the currentOp.
+   *
+   * @param {Integer} includeAll - if true also list currently idle operations in the result.
+   * @param {Function} callback - The callback.
+   */
+  currentOp(includeAll, callback) { // TODO: deal with db versions? && do we want operations arg?
+
+    this.database.admin().command({'currentOp': 1, '$all': includeAll}, (error, result) => {
```

Comment:
We don't have a normal style guide that I know of but as you found out, eslint will complain if you violate the rules we set up, and runs always in the travis build and has a pre commit hook.